lib/puppet-lint/plugins/check_unsafe_interpolations.rb

```diff
@@ -117,4 +117,13 @@ def get_exec_titles
   def interpolated?(token)
     INTERPOLATED_STRINGS.include?(token.type)
   end
+
+  # Finds the index of the next instance of `value` in `array` from the `current_index`
+  def colon_index_offset(tokens_slice, value)
```

Comment:
How about calling this function `index_offset_for` given that it is not limited to just colons.

Also is the description still correct now that we have changed what the function is doing?

Reply:
yeah good shout. And yes description will need updating. Cheers